Automattic/jetpack
# AI Assistant: Fix auto scrolling on iframe editor and Safari

Fixes the auto scrolling inside `iframe` editor (newest Gutenberg versions) and Safari (Desktop and Mobile).

## Proposed changes:

* Rewrite the logic to a single call for `snapToBottom`.
* Rewrite the logic to detect user scroll by add and removing `eventListener` when doing auto scroll.
* Use `scrollIntoView` to make cross-browser better.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Test it with embeded editor and iframe editor (newest Gutenberg versions).
* It should start the `auto-scroll` and as soon you scroll, it should stop the auto scroll.
* Make sure you test it on Safari (Desktop and Mobile).
